pyramid_orb/utils.py

```diff
@@ -88,7 +88,8 @@ def get_context(request, model=None):
 
         # extract payload items
         for key, value in param_values.items():
-            schema_object = model.schema().column(key, raise_=False) or model.schema().collector(key)
```

Comment:
this approach was too limited, it doesn't support the new feature of custom filters.  Instead of looking through the full path, it really just needs to validate the root key to know that it starts at a column or collector to be related to the model that is being talked to